lib/idempo/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 class Idempo
-  VERSION = "1.2.2"
+  VERSION = "1.2.3"
```

Comment:
**thought:** This version increase seems a bit confusing to me. We're delivering rack 3 compatibilities, but we're incrementing only patch version. 

This improvement, in my mind, warrants 2.0 or at least 1.3 version.

Reply:
Well, from the semver angle I figured
* We are not adding extra features (maybe Rack 3 compatibility with non-rewindable requests can be seen as one, but there are next to no uses of this in the wild) - so why bump the minor version?
* We are not breaking any functionality nor are we changing APIs and not introducing breaking changes - so no bump needed for the major version either

We could bump the minor version though.

Reply:
Sounds great to me.